.gitignore

```diff
@@ -1,3 +1,14 @@
+# Generated files
+/Make.include
+/autom4te.cache
+/config.h
+/config.h.in
+/config.log
+/config.status
+/configure
+/fireflies.spec
+/installit
+
```

Comment:
autom4te.cache, config.log, and config.status are the only files that can safely be deleted and still build. Even if the others can be autogenerated (some cannot - config.h.in?), I want to keep them checked in so you don't need autoconf to build.